targets/atmega2560.json

```diff
@@ -0,0 +1,20 @@
+{
+    "inherits": ["avr"],
+    "llvm-target": "avr-atmel-none",
+    "cpu": "atmega2560",
+    "build-tags": ["atmega2560"],
```

Comment:
Ideally the chip and the board would be separated, but they don't necessarily have to.

* If they are separated, there is one atmega2560.json that defines only the `atmega2560` and `atmega` build tag, and one arduino-mega2560.json that defines board-specific configuration (such as the `arduino_mega2560` build tag, the bootloader size, and the flash command).
* Otherise, this file should be named arduino-mega2560.json and also include the `arduino_mega2560` build tag.

Also, this chip is an ATmega so should define the `atmega` build tag.

Reply:
Ok I was wondering how this should be broken down and what the `target` should be named.

Reply:
I'll see if I can get this split up correctly!